patches/chrome-browser-ui-views-frame-multi_contents_view.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
+index 8ad29b28fa93b5a85f3cf832dcbead7f845c2893..eb72fe3f967dc121ec085b5369e7867f738d1f91 100644
+--- a/chrome/browser/ui/views/frame/multi_contents_view.h
++++ b/chrome/browser/ui/views/frame/multi_contents_view.h
+@@ -17,7 +17,7 @@
+ 
+ class BrowserView;
+ class ContentsWebView;
+-class MultiContentsResizeArea;
++// class MultiContentsResizeArea;
```

Comment:
This redefine `MultiContentsResizeArea views::ResizeArea` makes this build error.